data/styles/clear/include/Icons.mapcss

```diff
@@ -1175,9 +1194,14 @@ area|z16[historic=wayside_shrine]
 
 node|z17-[historic=boundary_stone],
 area|z17-[historic=boundary_stone],
-node|z17-[historic=wayside_cross],
-area|z17-[historic=wayside_cross]
+node|z17-[historic=gallows],
+area|z17-[historic=gallows],
+node|z17-[historic=pillory],
+area|z17-[historic=pillory]
 {icon-image: monument-m.svg;font-size: 11;icon-min-distance: 8;}
+node|z17-[historic=wayside_cross],
+area|z17-[historic=wayside_cross],
+{icon-image: cross-m.svg;font-size: 11;icon-min-distance: 8;}
```

Comment:
Strange why this historic styles was distinguished on z16 and z17-, if they can be combined into z16-
I'd also make gallows and pillory from z16 like all other historic-* stuff.